ocean_lib/aquarius/test/test_aquarius.py

```diff
@@ -51,9 +51,7 @@ def test_aqua_functions_for_single_ddo(
     data_provider = DataServiceProvider
     file1_dict = {"type": "url", "url": "https://url.com/file1.csv", "method": "GET"}
     file1 = FilesTypeFactory(file1_dict)
-    encrypt_response = data_provider.encrypt(
-        [file1], "http://172.15.0.4:8030/api/services/encrypt"
-    )
+    encrypt_response = data_provider.encrypt([file1], config.provider_url)
```

Comment:
I don't think we need the provider_url. In this test, the provider will fallback onto the config provider_url anyway.